docker-compose-integration-test.yml

```diff
@@ -17,3 +17,14 @@ services:
     ports:
       - "5900:5900"
     shm_size: 2g
+  rust-sdk:
+    image: b4handjr/nimbus-rust-image:latest
```

Comment:
I like the idea of a dockerfile...I kinda want application-services to have a dockerfile that we could use as mine is very specific to nimbus (it builds ```nimbus.py```). I could certainly put it in this repo and we maintain it here since it is just for us.

Reply:
Ooo I like that idea. I think there is a small decision to make around how versioning would work (app services versions vs app versions when it comes to testing)

once that's a little clearer we can have the dockerfile in app services and have it rebuild on a regular cadence, whether it's cron-job, on release, etc

Reply:
But for now, I'm happy with this 😄